eZ Platform/Admin UI/Field Type Edit/Requirements.md

```diff
@@ -0,0 +1,45 @@
+# Content editing
+
+
+# Objective
+
+"As an eZ Engineer, I want...
+
+To create a functional, experimental implementation of a refactoring of the Field Type API in eZ Platform supporting full CRUD (create, read, update and delete) for content creation and editing. The purpose of the research is to find a common way to define the user interface using different technologies."
+
+Underlying goals are:
+
+* Simplify how to create new custom Field Type
+* Reduce errors by limiting the number of places UI definitions are done
```

Comment:
I think the implied scope here is the whole Field Type, as in:
- Value
- Type
  - Potentially storage handler
  - And currently legacy db converter
- symfony forms mapper
  - + potentially data transformer
- yui service for edit
  - + handlebars template for edit
- yui service for view
  - + handlebars template for view
- twig block for frontend view
  - + potentially helpers/custom-fieldtype-service to help with view needs

And then there is indexer for and potentially advance field name handling.